doc/src/manual/running-external-programs.md

```diff
@@ -307,7 +307,18 @@ IO redirection can be accomplished by passing keyword arguments `stdin`, `stdout
 ```julia
 pipeline(`do_work`, stdout=pipeline(`sort`, "out.txt"), stderr="errs.txt")
 ```
+For example, you can pass a `IOBuffer`, and later read from it, after resetting the stream back to its beginning.
```

Comment:
```suggestion

For example, you can pass a `IOBuffer`, and later read from it, after resetting the stream back to its beginning:

```

I'm not sure where (if it is mentioned), but you can read from a `Cmd` directly too:

```
julia> lines = eachline(`ls /`)
Base.EachLine{Base.PipeEndpoint}(Base.PipeEndpoint(RawFD(23) open, 0 bytes waiting), Base.var"#882#883"{Base.Process}(Process(`ls /`, ProcessRunning)), false)

julia> collect(lines)
20-element Vector{String}:
 "bin"
 "boot"
 "dev"
 "etc"
 "grub"
 "home"
 "lib"
 "lib64"
 "lost+found"
 "mnt"
 "opt"
 "proc"
 "root"
 "run"
 "sbin"
 "srv"
 "sys"
 "tmp"
 "usr"
 "var"
```

If it's not mentioned yet, it would be good to add it here too.

Reply:
Definitely useful. I didn't know that.